app/components/common/CustomModal.tsx

```diff
@@ -49,80 +51,104 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     color: 'gray',
   },
-  firstBtnStyle: {
-    flex: 1,
+  BtnListStyle: {
```

Comment:
- 대문자로 시작하는건 좋지 않습니다.
- 또 굳이 Style 을 붙이는것도 좋지 않습니다.
```suggestion
 bntList: {
```